language/stdlib/transaction_scripts/modify_publishing_option.move

```diff
@@ -0,0 +1,5 @@
+use 0x0::ScriptWhitelist;
+
+fun main(args: vector<u8>) {
```

Comment:
Comment explaining what this script does and who has the authority to use it?

Reply:
> Very cool!
> 
> A question, though: would it make sense to add a test for updating the whitelist (i.e., adding a new script)?

In this current test, I changed the option from whitelist mode to the custom script mode. Adding a script should be fine, the main problem is just how to obtain the raw bytes of the new script. Do you want me to add a test for adding empty main script to the whitelist?

Reply:
>  the main problem is just how to obtain the raw bytes of the new script. 

Ah, gotcha! I suppose you could have a link to the path of one of the `.mv`'s in `language/stdlib` and then hash that + use the script binary in your test to make sure both stay in sync.

I think it's important to have this test given that updating the whitelist is what we actually plan to do in prod.

Also, wondering about the raw bytes/Move value for on-chain configs question again... It seems like this might be easier to test if the whitelist was a Move value.

Reply:
> Ah, gotcha! I suppose you could have a link to the path of one of the .mv's in language/stdlib and then hash that + use the script binary in your test to make sure both stay in sync.

Let me backtrack on this approach--https://github.com/libra/libra/pull/3090 will insist that all of the `.mv`'s on disk are in the whitelist.